test/fixtures/linter/autofix/bigquery/006_fix_ignore_templating/after.sql

```diff
@@ -11,7 +11,8 @@ SELECT * EXCEPT({% include query %}) FROM
             FROM tbl1
             {% endif %}
         INNER JOIN tbl2
-            ON tbl1.the_name = tbl2.the_name
+            ON
+                tbl1.the_name = tbl2.the_name
```

Comment:
Possible to keep these on the same line?

I realize some of these questions are nitpicky. I wonder what the overall effect of these changes looks like on large, real-world SQL files. My main concern is if we're adding lots of additional lines. I once used a formatter that easily increased the number of lines 20-40%. Made the code "prettier" but actually much harder to read, because lots of scrolling.

Reply:
Yeah - this is another example of where I think an implicit indent might be appropriate. I'll add `ON` clauses into #4054 . I think being able to configure this one would be very useful. A newline there is _strictly_ the most correct, but I also grant that it feels like an unnecessary newline for many users.